benchmarks/README.md

```diff
@@ -0,0 +1,70 @@
+# Benchmarks
+
+## Table of Contents
+
+- [Benchmark Runs](#benchmark-runs)
+  - [performance.now()](#performancenow)
+  - [In An HTTP Server](#in-an-http-server)
+
+## Benchmark Runs
+
+### performance.now()
+
+The below benchmarks were run with a set number of items in each `Map`; and
+using `performance.now()` to check how long `.forEach()` method took to return a
+record. The benchmarks application searched for the last item in each `Map`.
+Below are the average times showing how long it took searches to complete and
+what method was used.
+
+Command used:
+
+```
+$ deno run -A benchmarks/app.ts [map|service] [number of seconds] [number of records to create]
+```
+
+#### Results
+
+```
+Performing search with 1,000 records(s) for 10s.
+Searching took an avg of 0.00002s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
+
+Performing search with 10,000 records(s) for 10s.
+Searching took an avg of 0.00014s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
+
+Performing search with 100,000 records(s) for 10s.
+Searching took an avg of 0.00136s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
+
+Performing search with 1,000,000 records(s) for 10s.
+Searching took an avg of 0.01379s using Map.forEach().
+Searching took an avg of 0.00001s using IndexService.search().
```

Comment:
what's the time on _just_ the first hit, so without memoization? haven't looked into Moogle thoroughly, but what does moogle do that `Map.forEach` couldn't do with a simple memo wrapper? i guess what i'm saying is that maybe the tests would be more telling if it were more than a string search. Idk, what do you think?

Reply:
agreed. we should show the output of a single search to fill the gap of "what's the time on the first search in moogle compared to the time on map.foreach call"

Reply:
not strong feelings though... the package seems like it's doing what it is intended for.